nacc_attribute_deriver/schema/operation.py

```diff
@@ -45,12 +45,10 @@ def get_list_type(expression_type: type) -> type:
 
 def get_date_tagged_type(expression_type: type) -> type:
     if hasattr(expression_type, "__pydantic_generic_metadata__"):
-        origin = expression_type.__pydantic_generic_metadata__["origin"]  # type: ignore
+        origin = expression_type.__pydantic_generic_metadata__["origin"]
```

Comment:
We lost the type ignore pragmas in this file. They are all things we can't do anything about, so I'd like to keep them